README.md

```diff
@@ -54,4 +54,47 @@ pelfs = PelicanFileSystem(“some-director-url”)
 file1 = PelicanMap(“/namespace/file/1”, pelfs=pelfs)
 file2 = PelicanMap(“/namespace/file/2”, pelfs=pelfs)
 ds = xarray.open_mfdataset([file1,file2], engine='zarr')
-```
\ No newline at end of file
+```
+
+### Specifying Endpoints
+
+The following describes how to specify endpoints to get data from, rather than letting PelicanFS and the director determine the best cache. PelicanFS allows you to specify whether to read directly from the origin (bypassing data staging altogether) or to name a specific cache to stage data into. 
+
+**Note**
+> If both direct reads and a specific cache are set, PelicanFS will use the specified cache and ignore the direct reads setting.
+
+
+#### Enabling Direct Reads
+
+Sometimes you might wish to read data directly from an origin rather than via a cache. To enable this at PelicanFileSystem creation, just pass in `dirReads=True` to the constructor.
+
+```python
+pelfs = PelicanFileSystem("some-federation-url", dirReads=True)
+```
+
+Direct reads can also be turned on an off using the class function `set_direct_reads` like so:
+
+```python
+pelfs.set_direct_reads(True)
+hello_world_from_origin = pelfs.cat('/ospool/uc-shared/public/OSG-Staff/validation/test.txt')
+```
+
+#### Specifying a Cache
+
+If you want to specify a specific cache to stage your data into (as opposed to the highest priority working cache), this can be done by passing in a cache URL during PelicanFileSystem construction via the `specifiedCacheUrl` variable:
+
+```python
+pelfs = PelicanFileSystem("some-federation-url", specifiedCacheUrl="some-cache-url")
+```
+
+Or by setting it via the `set_cache` function like so:
+
+```python
+pelfs.set_cache("some-cache-url")
```

Comment:
Same as above -- let's look around at a few other fsspec implementations and see if others provide getters and setters like this.  If it's not common, let's skip it for now.

(After all, once it's in the public API, it's somewhat there forever!)